thoth/glyph/cli.py

```diff
@@ -87,5 +88,23 @@ def generate(output: str) -> None:
             output_file.write("some output generated to file\n")
         _LOGGER.info("Generated output stored in %r", output)
 
+@cli.command()
+@click.option(
+    "--message",
+    "-m",
+    type=str,
+    default="-",
+    help="Commit message to be classified",
+)
+def classify(message: str) -> None:
+    """Generate CHANGELOG entries from the current Git project."""
+    _LOGGER.info("Hello, glyph!")
+    if message == "-":
+        click.echo("Please enter message")
```

Comment:
You can remove the `default` in `@click.option` and make it `required=True` to avoid this check.